k8_cortx_cloud/destroy-cortx-cloud.sh

```diff
@@ -132,90 +134,93 @@ done <<< "$(kubectl get namespaces)"
 
 num_motr_client=$(extractBlock 'solution.common.motr.num_client_inst')
 
+function uninstallHelmChart()
+{
+    local chart=$1
+    local ns=()
+    [[ -n ${2:-} ]] && ns+=("--namespace=$2")
+    # Silence "release: not found" messages as those can be expected
+    helm uninstall "${chart}" "${ns[@]}" 2> >(grep -v 'uninstall.*release: not found')
```

Comment:
Doing some test runs on deploy/destroy loops and catching errors on this command when run it attempts to uninstall the cortx-platform chart.

```bash
########################################################
# Delete Persistent Volumes                            #
########################################################
pvc-1a3c8c01-0e34-4338-9517-12df189811bf
pvc-7acbe83e-3226-4cb0-8b95-71eccbb5a089
pvc-946f9683-90bb-4e62-b5c0-fb9e3599853c
Removing pvc-1a3c8c01-0e34-4338-9517-12df189811bf
persistentvolume "pvc-1a3c8c01-0e34-4338-9517-12df189811bf" deleted
Removing pvc-7acbe83e-3226-4cb0-8b95-71eccbb5a089
persistentvolume "pvc-7acbe83e-3226-4cb0-8b95-71eccbb5a089" deleted
Removing pvc-946f9683-90bb-4e62-b5c0-fb9e3599853c
persistentvolume "pvc-946f9683-90bb-4e62-b5c0-fb9e3599853c" deleted
########################################################
# Delete Cortx Kubernetes Prereqs                      #
########################################################
./destroy-cortx-cloud.sh: line 143: ns[@]: unbound variable
```

Reply:
The deploy script exits immediately from this error and leaves the deployment in a somewhat broken state, so I needed to manually clean a bunch of things up both on k8s and in the local fs before I could deploy again.

Reply:
I can't reproduce this error. Could you run the scripts with `bash -x` and send/attach the output?

Reply:
The most recent run shows the following:
```
Removing pvc-fa8f7c99-c823-431c-a635-b988d1b32d6f
+ kubectl delete pv pvc-fa8f7c99-c823-431c-a635-b988d1b32d6f
persistentvolume "pvc-fa8f7c99-c823-431c-a635-b988d1b32d6f" deleted
+ [[ default != \d\e\f\a\u\l\t ]]
+ deleteKubernetesPrereqs
+ printf '########################################################\n'
+ printf '# Delete Cortx Kubernetes Prereqs                      #\n'
########################################################
+ printf '########################################################\n'
+ uninstallHelmChart cortx-platform
+ local chart=cortx-platform
# Delete Cortx Kubernetes Prereqs                      #
########################################################
+ ns=()
+ local ns
+ [[ -n '' ]]
./destroy-cortx-cloud.sh: line 143: ns[@]: unbound variable
[root@ssc-vm-g2-rhev4-1987 k8_cortx_cloud]# pwd
/root/deploy-scripts/k8_cortx_cloud
```

Reply:
Ok, I'll revert the `set -u`. This appears to be a problem in bash that was fixed in Bash 4.4+ or so. Locally I'm using Bash 5, the lab servers are using 4.2. 😢

Reply:
> Doing some test runs on deploy/destroy loops and catching errors on this command when run it attempts to uninstall the cortx-platform chart.
> 
> ```shell
> ########################################################
> # Delete Persistent Volumes                            #
> ########################################################
> pvc-1a3c8c01-0e34-4338-9517-12df189811bf
> pvc-7acbe83e-3226-4cb0-8b95-71eccbb5a089
> pvc-946f9683-90bb-4e62-b5c0-fb9e3599853c
> Removing pvc-1a3c8c01-0e34-4338-9517-12df189811bf
> persistentvolume "pvc-1a3c8c01-0e34-4338-9517-12df189811bf" deleted
> Removing pvc-7acbe83e-3226-4cb0-8b95-71eccbb5a089
> persistentvolume "pvc-7acbe83e-3226-4cb0-8b95-71eccbb5a089" deleted
> Removing pvc-946f9683-90bb-4e62-b5c0-fb9e3599853c
> persistentvolume "pvc-946f9683-90bb-4e62-b5c0-fb9e3599853c" deleted
> ########################################################
> # Delete Cortx Kubernetes Prereqs                      #
> ########################################################
> ./destroy-cortx-cloud.sh: line 143: ns[@]: unbound variable
> ```

As of b66092e, I am no longer seeing this issue. All good on my end.